dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -349,17 +348,14 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 									data-print-layout="hide"
 									element="aside"
 								>
-									<MostViewedFooterLayout>
-										<MostViewedFooter
-											tabs={[
-												{
-													trails: trails.slice(10),
-												},
-											]}
-											sectionName="Most viewed"
-											// TODO: Include mostCommented & mostShared once we have this data in the FE response
-										/>
-									</MostViewedFooterLayout>
+									<FrontMostViewed
+										displayName={collection.displayName}
+										trails={trails.slice(0, 10)}
```

Comment:
Would we want to leave the slicing logic to the container as it knows that it can only take 10? It colocates the logic with where it's rendered, and not sure the instantiating component should care.

Reply:
+1 turns out we're slicing it in the FrontMostViewed container anyway :)